packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -490,6 +490,15 @@ abstract final class SystemChrome {
     );
   }
 
+  static void Function(SystemUiOverlayStyle? style)? _systemUiOverlayStyleCallback;
+
+  /// Sets the callback method for responding to changes in the SystemUiOverlayStyle.
+  // ignore: use_setters_to_change_properties
```

Comment:
Why this ignore? Ideally, we design the API without introducing these kind of ignores.